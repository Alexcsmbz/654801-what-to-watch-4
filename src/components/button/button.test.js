import Button from './button.jsx';

describe(`Button snapshot test`, () => {
  it(`Button should render button`, () => {
    const tree = renderer.create(
        <Button
          button={{
            name: `name`,
            onClick: () => {},
            className: `className`
          }}
          icon={{
            iconKey: `#icon-key`,
            width: `19`,
            height: `19`,
          }}
        />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
