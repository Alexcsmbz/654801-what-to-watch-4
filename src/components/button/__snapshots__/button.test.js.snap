// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button snapshot test Button should render button 1`] = `
<button
  className="catalog__button"
  onClick={[Function]}
  type="button"
>
  Show more
</button>
`;
